src/Pages/Home/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from 'react';
+import style from './Home.module.scss'
+import ScheduleRepositories from '../../Services/repositories/ScheduleRepositories';
+import LogoImage from '../../Components/Atoms/LogoImage';
+import Button from '../../Components/Atoms/Button';
+import Text from '../../Components/Atoms/Text';
+import { Input } from '../../Components/Atoms/Input/Input';
+
+export default function Home() {
+    const [isServiceOpen, setIsServiceOpen] = useState<boolean>();
+    const [closingTime, setClosingTime] = useState("");
+    const [openingTime, setOpeningTime] = useState("");
+    const [randomImage, setRandomImage] = useState("Harley");
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const data = await ScheduleRepositories.getSchedule();
+
+                setIsServiceOpen(data.isOpen);
+                setClosingTime(data.closingTime);
+                setOpeningTime(data.openingTime);
+            } catch (error) {
+                console.error(
+                    "Erro ao obter dados do horário de funcionamento:",
+                    error,
+                );
+            }
+        };
+        fetchData();
+    }, []);
+
+    const generateRandomName = () => {
```

Comment:
Eu vi que essa função gera a string para passarmos para a api da imagem certo? Eu acho que o nome não condiz com o que faz, que tal trocarmos para.

```suggestion
    const getRandomDiceImageString = () => {
```